omod/src/main/webapp/fragments/diagnosis/encounterDiagnoses.gsp

```diff
@@ -13,7 +13,7 @@
         {{ if (item.code) { }}
         {{- item.code }}
         {{ } else if (item.concept) { }}
-        ${ui.message("coreapps.consult.codedButNoCode")}
+        {{- item.preferredCodingSource }} ${ui.message("coreapps.consult.codedButNoCode")}
```

Comment:
And here something like (this has to be checked):
```gsp
${ui.message("coreapps.consult.codedButNoCode", item.preferredCodingSource)}
```

Reply:
This is quite impossible to do because `item.preferredCodingSource` is javascript that is referred to after `ui.message()` has done its part. Maybe such an usage as passing `preferredCodingSource` through the fragment's `config` to do the same such as 

`${ui.message("coreapps.consult.codedButNoCode", config.preferredCodingSource)}`

Just a thought. But let me check this out.

Reply:
So @mks-d, the above approach using `item.preferredCodingSource` fails with a [pre-mature error](https://hastebin.com/erudamebon.coffeescript). I've just made a branch of this one with working code [here](https://github.com/Ruhanga/openmrs-module-coreapps/pull/2/commits/9637440cce3a68d804f7a8a6f4e7b6df381b9382). So I should go ahead with this approach. And to make the solution uniform, I'd have to refactor the changes here so the `preferredCodingSource` is acually placed within `<input id="diagnosis-search . . . . ./>"` like was similarly done on RA-1603.

Reply:
If this works, let's settle on that:
```java
${ui.message("coreapps.consult.codedButNoCode", config.preferredCodingSource)}
```